modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/modal/transition/bulk/BulkTransitionModal.es.js

```diff
@@ -456,8 +459,15 @@ describe('The BulkTransitionModal component should', () => {
 		fireEvent.click(nextBtn);
 	});
 
+	test('render a panel for each task names', () => {
+		const taskNamesPanel = document.querySelectorAll('h4.mt-2');
```

Comment:
@jotascript I think it would better to use the 'panel-header' class to form this selector, for readability reasons.